src/git-sync-all

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
+
+
+function preconditions {
+  fetch
+}
+
+
+function skip_message {
+  echo "To skip the sync of the '$(get_current_branch_name)' branch"
```

Comment:
This is a grammatically strange output. Please either improve it, or add a comment describing what exactly this method is for.